Makefile

```diff
@@ -124,6 +129,22 @@ registry-login:
 build-image:
 	ko publish --base-import-paths --tags="${IMAGE_TAG}" ./cmd/$(IMGCTRL) 
 
+# downloads and unpacks helm binary in output directory
+install-helm:
```

Comment:
to follow other entries we have in this Makefile we better name this as `get-helm` instead.